toolchest_client/tools/api.py

```diff
@@ -135,7 +135,7 @@ def kraken2(output_path, inputs=[], database_name="standard", database_version="
         >>> toolchest.kraken2(
         ...     tool_args="",
         ...     inputs="./path/to/input.fastq",
-        ...     output_path="./path/to/output.txt",
+        ...     output_path="./path/to/output.tar.gz",
```

Comment:
This should be modified, but it shouldn't be a .tar.gz. It should be a directory, e.g. `./path/to/output`.

The compression and decompression happens in the background. Try using the Kraken 2 tool to see how it works.

Overall, I think it's time to revamp our documentation.